docs/decisions/0007-executive-education-2U-terms-policies-user-profile-data-route.md

```diff
@@ -0,0 +1,74 @@
+# 0007. Add page route for gathering consent for terms/policies and user profile data for Executive Education (2U) courses
+
+## Status
+
+Accepted (08-01-2022)
+
+## Context
+
+As part of our Product Line Integration work (PLI) to make 2U content (i.e., Executive Education courses) accessible to customers with an external Learning Management System (LMS), we will need to collect required information from the user in order for fulfillment of Executive Education (2U) courses.
```

Comment:
grammer: "need to collect required information from the user ~~in order~~ for fulfillment"